modules/auxiliary/admin/networking/cisco_dcnm_download.rb

```diff
@@ -34,7 +34,7 @@ def initialize(info = {})
             [ 'CVE', '2019-1621' ],
             [ 'URL', 'https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-20190626-dcnm-bypass' ],
             [ 'URL', 'https://tools.cisco.com/security/center/content/CiscoSecurityAdvisory/cisco-sa-20190626-dcnm-file-dwnld' ],
-            [ 'URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/exploits/metasploit/cisco_dcnm_download.rb' ],
```

Comment:
Question but this is still a valid link, did you want to keep it and add in the TXT reference?